_data/sites.json

```
@@ -2997,6 +2997,19 @@
         ]
     },
 
+    {
+        "name": "Borussia Dortmund",
+        "url": "https://account.bvb.de/s/",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
If it needs email contact, we classify it as hard

---

Okay, thanks for the correction.